namer/dnssrv/src/test/scala/io/buoyant/namer/dnssrv/DnsSrvNamerTest.scala

```diff
@@ -0,0 +1,53 @@
+package io.buoyant.namer.dnssrv
+
+import com.twitter.finagle._
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.util.LoadService
+import com.twitter.util.{Duration, NullTimer}
+import io.buoyant.config.Parser
+import io.buoyant.namer.{NamerConfig, NamerInitializer}
+import io.buoyant.test.FunSuite
+import org.scalatest.Matchers
+import org.xbill.DNS
+
+class DnsSrvNamerTest extends FunSuite with Matchers {
+
+  test("sanity") {
+    // ensure it doesn't totally blowup
+    val _ = DnsSrvNamerConfig(
+      Some(15),
+      Some(List("localhost"))
+    ).newNamer(Stack.Params.empty)
+  }
+
+  test("service registration") {
+    assert(LoadService[NamerInitializer]().exists(_.isInstanceOf[DnsSrvNamerInitializer]))
+  }
+
+  test("parse config") {
+    val yaml = s"""
+                  |kind: io.l5d.dnssrv
+                  |experimental: true
+                  |refreshIntervalSeconds: 60
+                  |dnsHosts:
+                  |- localhost
+      """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(DnsSrvNamerInitializer)))
+    val curator = mapper.readValue[NamerConfig](yaml).asInstanceOf[DnsSrvNamerConfig]
+    assert(curator.refreshIntervalSeconds === Some(60))
+    assert(curator.dnsHosts === Some(Seq("localhost")))
+  }
+
+  test("can resolve some public SRV revord") {
```

Comment:
I'd feel somewhat more comfortable if tests that create external network connections went in integration rather than unit tests.